packages/compartment-mapper/test/fixtures-policy/node_modules/myattenuator/index.js

```diff
@@ -0,0 +1,7 @@
+export const attenuate = (params, originalModuleNamespace) => {
+  const ns = params.reduce((acc, k) => {
+    acc[k] = originalModuleNamespace[k];
+    return acc;
+  }, {});
+  return ns;
```

Comment:
Consider freezing.

Reply:
Ok, I'm freezing. It's not necessary though, because it's not the final namespace - it geets Object.assign-ed to exportsProxy in execute.